### PR TITLE
Alex tutorial changes

### PR DESCRIPTION
**Pull request by loftusa**
PRing this!

- Added a check for FSL and AFNI into `ndmg_bids`
- Streamlined the tutorial a bunch

- Fiddled with the TRAVIS matrix a bit to get it to check deterministic/probablistic tractography as well

### CODE REVIEW

**Review by dPys — APPROVED** [review]

**Comment by dPys** [comment]
Added my review

File: Dockerfile [hunk 1]
```diff
@@ -64,10 +64,10 @@ RUN \
     pip3.6 install numpy
 
 RUN \
-    pip3.6 install nibabel scipy python-dateutil pandas boto3 awscli matplotlib nilearn sklearn pandas cython vtk pyvtk fury awscli requests ipython duecredit graspy scikit-image
+    pip3.6 install nibabel scipy python-dateutil pandas boto3 awscli matplotlib nilearn sklearn pandas cython vtk pyvtk fury awscli requests ipython duecredit graspy scikit-image networkx
```
Comment (Lawreros) [inline Dockerfile:67]: Get rid of repeated 'pandas' and 'awscli'

Comment (Lawreros) [inline Dockerfile]: remove repeated pandas and awscli

**Review by Lawreros — DISMISSED** [review]
Looks good, as far as I can tell, only real comment is installing things twice in the Dockerfile

**Pull request merged**